libdnf/module/module_sack.cpp

```diff
@@ -295,6 +316,26 @@ void ModuleSack::Impl::module_filtering() {
     // dnf_sack_set_module_includes(sack, includeQuery.getResultPset());*/
 }
 
+
+void ModuleSack::Impl::make_provides_ready() {
```

Comment:
I have one additional suggestion related to make provides ready:
I recommend to add in `module_sack.cpp` two additional lines:
In `void ModuleSack::add()` and `void ModuleSack::Impl::add_modules_without_static_context()` to set `provides_ready = false` because additional data will also require to make provide ready.